### PR TITLE
repo and product name change

### PR DESCRIPTION
**Pull request by joe-lipson**
As well as changing the repo and application name all the resource names and descriptions have been updated as well. It will mean that this first update will not be able to be done in place, The old environment will have to be taken down then the new one launched.

There's a bunch of other commits that were on the testing branch that will go out with this change that will need to be reviewed

### CODE REVIEW

File: src/components/client-registration.tsx [hunk 1]
```diff
@@ -96,7 +96,7 @@ const  ClientRegistration: React.FC<{config: AuthConfig, updateRegistrationSucce
     const [resourceOwner, setResourceOwner] = useState("");
     const [resourceOwnerEmail, setResourceOwnerEmail] = useState("");
     const [additionalInfo, setAdditionalInfo] = useState("");
-    const [scopes, setScopes] = useState("openid email profile roles"); 
+    const [scopes, setScopes] = useState("openid email profile ala/roles");
```
Comment (yasima-csiro) [inline src/components/client-registration.tsx:99]: This change needs to be done once the cognito is released? Until that token generation happens with CAS.

File: src/components/client-registration.tsx [hunk 2]
```diff
@@ -106,7 +106,7 @@ const  ClientRegistration: React.FC<{config: AuthConfig, updateRegistrationSucce
             "appName": appName,
             "callbackUrl": callbackUrl ? callbackUrl + ', https://tokens.ala.org.au': 'https://tokens.ala.org.au',
             "resourceOwner": resourceOwner,
-            "scopes": scopes ? scopes: 'openid email profile roles',
+            "scopes": scopes ? scopes: 'openid email profile ala/roles',
```
Comment (yasima-csiro) [inline src/components/client-registration.tsx:109]: This change needs to be done once the cognito is released? Until that token generation happens with CAS.

File: src/helpers/config.ts [hunk 3]
```diff
@@ -13,6 +14,7 @@ const config: AuthConfig = {
   client_secret: import.meta.env.VITE_CLIENT_SECRET,
   scope: import.meta.env.VITE_SCOPE,
   authority: import.meta.env.VITE_OIDC_AUTHORITY,
+  cognito_logout_uri: import.meta.env.VITE_COGNITO_LOGOUT_URI,
```
Comment (yasima-csiro) [inline src/helpers/config.ts:17]: Setting up this is fine. But until cognito is released, this should work with CAS.

**Comment by joe-lipson** [comment]
Closing this as the the cognito changes can not be merged to production yet. The repo and name change updates have been merged directly to main

**Pull request closed without merging**